packages/slate-tables/src/core/onKeyDown.ts

```diff
@@ -86,27 +91,32 @@ function onDownPress(editor: TablesEditor): Point | undefined {
 
     const { activeCell, matrix } = traverse;
 
-    if (activeCell.row.isLast) {
-        return Editor.after(editor, matrix.path, { unit: 'block' });
-    }
-
-    const [, lastNodePath] = Editor.last(editor, activeCell.path);
-
-    if (!Path.equals(editor.selection.anchor.path, lastNodePath)) {
-        return undefined;
-    }
-
-    const { cellBelow } = activeCell;
-
-    if (cellBelow) {
-        return EditorCommands.findLeafPoint(
-            editor,
-            {
-                path: cellBelow.path,
-                offset: editor.selection.anchor.offset,
-            },
-            'highest',
-        );
+    const [, cellEnd] = Editor.edges(editor, activeCell.path);
```

Comment:
Same as above. It doesn't make sense to use `Editor.edges()` and only keep a part of the result.

Use `Editor.end()` directly.